src/utility.cpp

```diff
@@ -622,9 +622,9 @@ void setupVDOutput(Prismatic::Parameters<PRISMATIC_FLOAT_PRECISION> pars, const
 		dim2_name.write(strdatatype, dim2_name_str);
 		dim3_name.write(strdatatype, dim3_name_str);
 
-		const H5std_string dim1_unit_str("[n_m]");
-		const H5std_string dim2_unit_str("[n_m]");
-		const H5std_string dim3_unit_str("[mrad]");
+		const H5std_string dim1_unit_str("[Å]");
+		const H5std_string dim2_unit_str("[Å]");
+		const H5std_string dim3_unit_str("[rad]");
```

Comment:
This looks to be related to the fact that pars.detectorAngles is in units of radians, correct? Perhaps we should just multiply the detector angles array out by 1000, I would think that more people would prefer to reference mrads.

here: https://github.com/prism-em/prismatic/blob/126dcf3f023f8bdccfa553f68bd2b7837eac5310/src/Multislice_calcOutput.cpp#L144-L154

and here: 
https://github.com/prism-em/prismatic/blob/126dcf3f023f8bdccfa553f68bd2b7837eac5310/src/PRISM03_calcOutput.cpp#L108-L127

Reply:
I have changed to `pars.detectorAngles` to be in `mrad` and changed the rest accordingly.